src/auth.ts

```diff
@@ -96,148 +104,184 @@ export type ServiceInfo = {
-      return 'QuickBooks';
-    case 'salesforce':
-      return 'Salesforce';
-    case 'slack':
-      return 'Slack';
-    case 'spotify':
-      return 'Spotify';
-    case 'stripe':
-      return 'Stripe';
-    case 'trello':
-      return 'Trello';
-    case 'twilio':
-      return 'Twilio';
-    case 'twitter':
-      return 'Twitter';
-    case 'twitch-tv':
-      return 'Twitch';
-    case 'ynab':
-      return 'You Need a Budget';
-    case 'youtube':
-      return 'YouTube';
-    case 'zeit':
-      return 'Vercel';
-    case 'zendesk':
-      return 'Zendesk';
-    default:
-      return service;
+function friendlyServiceName(service: Service): string {
+  if (service.type === 'service') {
+    switch (service.service) {
+      case 'adroll':
+        return 'Adroll';
+      case 'asana':
+        return 'Asana';
+      case 'box':
+        return 'Box';
+      case 'dev-to':
+        return 'Dev.to';
+      case 'dribbble':
+        return 'Dribbble';
+      case 'dropbox':
+        return 'Dropbox';
+      case 'contentful':
+        return 'Contentful';
+      case 'eggheadio':
+        return 'Egghead.io';
+      case 'eventil':
+        return 'Eventil';
+      case 'facebook':
+        return 'Facebook';
+      case 'firebase':
+        return 'Firebase';
+      case 'github':
+        return 'GitHub';
+      case 'gmail':
+        return 'Gmail';
+      case 'google':
+        return 'Google';
+      case 'google-ads':
+        return 'Google Ads';
+      case 'google-analytics':
+        return 'Google Analytics';
+      case 'google-calendar':
+        return 'Google Calendar';
+      case 'google-compute':
+        return 'Google Compute';
+      case 'google-docs':
+        return 'Google Docs';
+      case 'google-search-console':
+        return 'Google Search Console';
+      case 'google-translate':
+        return 'Google Translate';
+      case 'hubspot':
+        return 'Hubspot';
+      case 'intercom':
+        return 'Intercom';
+      case 'mailchimp':
+        return 'Mailchimp';
+      case 'meetup':
+        return 'Meetup';
+      case 'netlify':
+        return 'Netlify';
+      case 'product-hunt':
+        return 'Product Hunt';
+      case 'quickbooks':
+        return 'QuickBooks';
+      case 'salesforce':
+        return 'Salesforce';
+      case 'slack':
+        return 'Slack';
+      case 'spotify':
+        return 'Spotify';
+      case 'stripe':
+        return 'Stripe';
+      case 'trello':
+        return 'Trello';
+      case 'twilio':
+        return 'Twilio';
+      case 'twitter':
+        return 'Twitter';
+      case 'twitch-tv':
+        return 'Twitch';
+      case 'ynab':
+        return 'You Need a Budget';
+      case 'youtube':
+        return 'YouTube';
+      case 'zeit':
+        return 'Vercel';
+      case 'zendesk':
+        return 'Zendesk';
+      default:
+        return service.service;
+    }
+  } else {
+    return 'service identified by field ' + service.graphQLField;
```

Comment:
Let's just return `service.graphQLField` here or sentence cased `service.graphQLField` if you want to write a `sentenceCase` function. 

The end user that's looking at this may not be aware that there is any graphql thing anywhere.